.travis.yml

```diff
@@ -4,7 +4,8 @@ env:
     - TOXENV=py27
     - TOXENV=py34
 install:
-    - pip install -r requirements.txt coveralls flake8 sphinx testimony tox
+    - pip install -r requirements.txt coveralls flake8 sphinx tox
+    - pip install git+https://github.com/SatelliteQE/testimony.git
```

Comment:
A new Testimony version will be released soon and I will only unblock the PR to be merged after the new testimony version being released. That means that I will also undo this change.